Makefile

```diff
@@ -164,14 +180,19 @@ MACHINE_SIMPLE-$1=$$(MACHINE_SIMPLE-$$(ARCH-$1))
 else
 MACHINE_SIMPLE-$1=$$(ARCH-$1)
 endif
-SDK-$1=$$(basename $1)
+
+MACOSX_DEPLOYMENT_TARGET=10.6
```

Comment:
With this set to 10.6, the build fails.

Setting it to 10.8, the build succeeds and I verified that the produced python binary runs on Yosemite (in a VirtualBox VM).